commit 7571201cf7a3394e0dcd2b037aba1836089cc084
Author: Narthorn <narthorn@gmail.com>
Date:   Mon Oct 12 13:24:57 2015 +0200

    curses: Fix background transparency in terminal
    
    Patch comes from, and closes traviscross/mtr#72.

diff --git a/curses.c b/curses.c
index 02b7937..f95f5d1 100644
--- a/curses.c
+++ b/curses.c
@@ -700,13 +700,15 @@ void mtr_curses_open(void)
   initscr();
   raw();
   noecho(); 
+  int bg_col = 0;
   start_color();
 #ifdef HAVE_USE_DEFAULT_COLORS
-  use_default_colors();
+  if (use_default_colors() == OK)
+    bg_col = -1;
 #endif
   int i;
   for (i = 0; i < 8; i++)
-      init_pair(i+1, i, 0);
+      init_pair(i+1, i, bg_col);
 
   mtr_curses_init();
   mtr_curses_redraw();
